controller_interface/test/controller_base_test.cpp

```diff
@@ -0,0 +1,313 @@
+///////////////////////////////////////////////////////////////////////////////
+// Copyright (C) 2019
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//   * Redistributions of source code must retain the above copyright notice,
+//     this list of conditions and the following disclaimer.
+//   * Redistributions in binary form must reproduce the above copyright
+//     notice, this list of conditions and the following disclaimer in the
+//     documentation and/or other materials provided with the distribution.
+//   * Neither the name of Pal Robotics S.L. nor the names of its
```

Comment:
If Pal Robotics owns the Copyright maybe it should be clear in line 2?

Reply:
good point! I'll add some suggestions